network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html

```diff
@@ -1,41 +1,73 @@
-{% load wagtailimages_tags i18n %}
+{% load wagtailimages_tags i18n class_tags blog_tags wagtailroutablepage_tags %}
 
-<div class="row" id="home-read-more-news">
-    <div class="col-12">
-      <h2 class="h4-heading">{% trans "News you can use" %}</h2>
-      <span>{% trans "Read more news →" %}</span>
-    </div>
+<div class="row section-news-you-can-use section-margin">
+  <div class="col-12 d-flex flex-row justify-content-between">
+    <h2 class="h4-heading mb-4">{% trans "News you can use" %}</h2>
+    {# this should not be a hardcoded route... #}
```

Comment:
@Pomax do you remember why this should not be hardcoded?

Reply:
becca renamed `/blog` to something else and that broke everthing that ties into blogs. So we should really figure out a way to "not hardcode this". I've filed https://github.com/mozilla/foundation.mozilla.org/issues/5085 so we can fix that later.